queries/sql/highlights.scm

```diff
@@ -52,31 +56,40 @@
 [
  (keyword_asc)
  (keyword_desc)
- (keyword_external)
- (keyword_stored)
- (keyword_parquet)
- (keyword_csv)
- (keyword_rcfile)
- (keyword_textfile)
- (keyword_orc)
- (keyword_avro)
- (keyword_jsonfile)
- (keyword_temp)
- (keyword_cached)
- (keyword_uncached)
- (keyword_unlogged)
  (keyword_terminated)
  (keyword_escaped)
- (keyword_partitioned)
- (keyword_location)
- (keyword_lines)
  (keyword_unsigned)
  (keyword_nulls)
  (keyword_last)
+ (keyword_delimited)
+ (keyword_replication)
+ (keyword_auto_increment)
+ (keyword_default)
+ (keyword_collate)
+ (keyword_concurrently)
+ (keyword_engine)
+ (keyword_character)
+ (keyword_concurrently)
+ (keyword_preceding)
+ (keyword_following)
+ (keyword_first)
+] @attribute
+
+[
  (keyword_materialized)
  (keyword_temp)
  (keyword_temporary)
-] @attribute
+ (keyword_unlogged)
+ (keyword_external)
+ (keyword_parquet)
+ (keyword_csv)
+ (keyword_rcfile)
+ (keyword_textfile)
+ (keyword_orc)
+ (keyword_avro)
+ (keyword_jsonfile)
+ (keyword_sequencefile)
```

Comment:
Why are these storageclass? This is more to do with data manipulation

Reply:
Storage format. Makes sense to me.

Reply:
A little odd but sure..